### PR TITLE
[Elemental] Add Static Discharge Module

### PR DESCRIPTION
**Pull request by HawkCorrigan**

### CODE REVIEW

File: src/parser/shaman/elemental/modules/talents/StaticDischarge.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        major: 0.85,
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  suggestions(when: When) {
+    when(this.suggestionThresholds)
+      .addSuggestion((suggest, actual, recommended) =>
+        suggest(<span>Maximize your damage with Static Discharge by only using it with Flame Shock active.</span>)
```
Comment (MartijnHols) [inline src/parser/shaman/elemental/modules/talents/StaticDischarge.tsx]: Please see https://github.com/WoWAnalyzer/WoWAnalyzer/wiki/Suggestions for how to formulate suggestions.

**Review by MartijnHols — CHANGES_REQUESTED** [review]

File: src/parser/shaman/elemental/modules/talents/StaticDischarge.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  suggestions(when: When) {
+    when(this.suggestionThresholds)
+      .addSuggestion((suggest, actual, recommended) =>
+        suggest(
+          <span>Maximize your damage with Static Discharge by only using it with Flame Shock active on an enemy in range.
```
Comment (MartijnHols) [inline src/parser/shaman/elemental/modules/talents/StaticDischarge.tsx]: This is still missing the "Explain what was found" aspect. How about

`You missed ${formatPercentage(1 - actual)}% of the ticks of your <SpellLink id={SPELLS.STATIC_DISCHARGE.id} />. Try to maximize the ticks by only using it while Flame Shock is active on an enemy in range.`

Does it not hit anyone if Flame Shock is not active? [The tooltip](https://www.wowhead.com/spell=342243/static-discharge) suggests Flame Shock only indicates a preferred target, while the ticks will land on someone so long as they're in range. If that is the case I would make it:

`You missed ${formatPercentage(1 - actual)}% of the ticks of your <SpellLink id={SPELLS.STATIC_DISCHARGE.id} />. Try to land all ticks by only using it while you are sure there will be at least 1 enemy in range for the duration.`

Comment (HawkCorrigan) [reply src/parser/shaman/elemental/modules/talents/StaticDischarge.tsx]: Ye the tooltip is wrong.
I updated the suggestion.

**Review by Pewtro — APPROVED** [review]
Looks good to me with the wording now

**Comment by Pewtro** [comment]
Thanks!

**Pull request merged**